optuna/study.py

```diff
@@ -322,6 +322,31 @@ def _should_stop() -> bool:
     def set_user_attr(self, key: str, value: Any) -> None:
         """Set a user attribute to the study.
 
+        Please see the example code of the  :py:obj:`~optuna.study.Study.user_attrs`.
```

Comment:
How about using `.. seealso::` samely as other methods?
```suggestion
        .. seealso
        
            See :attr:`~optuna.study.Study.user_attrs` for related attribute.
```

Reply:
@HideakiImamura  Thanks for nice idea!